content/en/docs/userguide/use_kmeshctl.md

```diff
@@ -0,0 +1,239 @@
+---
+draft: false
+linktitle: Use kmeshctl
+menu:
+  docs:
+    parent: user guide
+    weight: 20
+title: Use kmeshctl
+toc: true
+type: docs
+
+---
+
+### Installation
+
+1. Installing from release binaries
+
+Pre-built binaries are available on our [releases page](https://github.com/kmesh-net/kmesh/releases).
+
+```bash
+# For AMD64 / x86_64
+[ $(uname -m) = x86_64 ] && curl -Lo ./kmeshctl https://github.com/kmesh-net/kmesh/releases/download/v0.5.0/kmeshctl-linux-amd64
+# For ARM64
+[ $(uname -m) = aarch64 ] && curl -Lo ./kmeshctl https://github.com/kmesh-net/kmesh/releases/download/v0.5.0/kmeshctl-linux-arm64
+chmod +x ./kmeshctl
+sudo mv ./kmeshctl /usr/local/bin/kmeshctl
+```
+
+2. Installing from source
+
+kmeshctl is still in rapid development. If you want to try latest features, you can directly build and install it from source.
+
+```bash
+# Clone source code from github
+git clone https://github.com/kmesh-net/kmesh.git
+
+# Build and install kmeshctl
+cd kmesh/
+make kmeshctl
+chmod +x ./kmeshctl
+sudo mv ./kmeshctl /usr/local/bin/kmeshctl
+```
+
+### Usage
+
+#### kmeshctl accesslog
+
+Enable or disable Kmesh's accesslog
+
+```bash
+kmeshctl accesslog [flags]
+```
+
+Examples:
+
+```bash
+# Enable Kmesh's accesslog:
+kmeshctl accesslog <kmesh-daemon-pod> enable
```

Comment:
<kmesh-daemon-pod> is optional, if ommitted, all accesslog enabled

Reply:
@LiZhenCheng9527 may need to optimize